libs/ui-toolkit/src/components/icon/vega-icons/svg-icons/icon-man.tsx

```diff
@@ -0,0 +1,27 @@
+export const IconMan = ({ size = 14 }: { size: number }) => {
+  return (
+    <svg
```

Comment:
Can you tidy all the svgs

- no xmlns
- no unnecessary fill/stroke attributes
- no hardcoded colors

Reply:
updated